### DISCUSSION TITLE
New style

### DISCUSSION THREAD

**Discussion by ItzAlmaZ** [BEEmod/BEE2-items#4134] [Feature Suggestions]
I understand that it's not very easy to make a style, but you can make a 3d skybox with supports and some buildings like other test chambers. (In simplified form (box). Around the test chamber there is a skybox with supports that go into the distance and we see them through a bottomless pit (water) and blocks of non-portal-suitable surfaces destroyed (automatically or by a separate block), This can be seen in Chapter 8. Most likely this is not feasible
![2022-05-01_12-49-57](https://user-images.githubusercontent.com/104623098/166137157-b2ce536e-7d55-41f7-b9bc-f50627635e1d.png)
![2022-05-01_12-51-31](https://user-images.githubusercontent.com/104623098/166137225-045d035a-733a-4af7-a150-a9ac93aff038.png)

**Comment by FrozenDragon0** [comment]
Something like this would be possible with the chamber exterior system.

**Comment by SimpleFoxOfficial** [comment]
this is cool and it was in bts style skybox. Sadly that they deleted this.
Also the destroyed non portal surfaces can be just the OBS room

**Reply by FrozenDragon0** [reply]
@Li7i4ka It is not deleted, once the chamber exterior system is implemented then the bottomless pits will return.
